projects/ngx-location-viewer/src/lib/components/ngx-location-viewer.component.ts

```diff
@@ -517,7 +533,7 @@ export class NgxLocationViewerComponent implements OnInit, OnChanges, OnDestroy
                                 const address = x.addressDetail[0];
                                 const marker = this.leafletMap.addHtmlMarker(
                                     address.addressPosition.wgs84,
-                                    this.leafletMap.getHtmlMarker('#000000', 'ai-location-target-1', '16px', { top: '-3px', left: '2px' }),
+                                    this.leafletMap.getHtmlMarker('#000000', 'ai-pin', '16px', { top: '-3px', left: '2px' }),
```

Comment:
Can you also fix this color in some way?

Reply:
What do you mean by 'fix'? Do you mean to set all that styling into a class? Then, I don't think so:

![image](https://github.com/digipolisantwerp/location-viewer_widget_angular/assets/46924202/51a400af-5dfc-41d8-8eca-21267ff00108)


Or if you would know a way, I would be happy to hear it :)

Reply:
Sorry, I was indeed not so clear. I would only make sure the marker uses the $THEME1-600 color instead of hardcoded black.

Reply:
**$THEME1-600** is not working? but **var(--THEME1-600)** does work.

But do we even want the 1-600? Because now we get this:

![image](https://github.com/digipolisantwerp/location-viewer_widget_angular/assets/46924202/8e880f9a-8d29-4657-8b98-10c1004600af)

Reply:
You are right that var(--THEME1-600) is the right property. Yes, we want to use this!

Reply:
Alright, the feedback implementation will be pushed in a minute, Thank you for your feedback!!